aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/Translator.java

```diff
@@ -178,15 +173,31 @@ static ListDomainsRequest translateToListRequest(final String nextToken) {
    * @param awsResponse the aws service describe resource response
    * @return list of resource models
    */
-  static List<ResourceModel> translateFromListRequest(final ListDomainsResponse awsResponse) {
+  static List<ResourceModel> translateFromListRequest(
+      final ListDomainsResponse awsResponse, final ResourceHandlerRequest<ResourceModel> request
+  ) {
+
     return streamOfOrEmpty(awsResponse.domains())
         .map(domain -> ResourceModel.builder()
+            .arn(buildArn("us-west-2", domain.owner(), domain.name()))
```

Comment:
why are we hardcoding us-west-2?

Reply:
`cfn test` does not override regions and defaults to us-west-2, I hardcoded this and forgot to remove it :(